sender-stackdriver/src/main/java/zipkin2/reporter/stackdriver/StackdriverSender.java

```diff
@@ -164,10 +164,39 @@ public void close() {
     ((ManagedChannel) channel).shutdownNow();
   }
 
-  final class PatchTracesCall extends UnaryClientCall<PatchTracesRequest, Empty> {
+  Span parseTraceIdPrefixedSpan(byte[] traceIdPrefixedSpan) {
+    // start parsing after the trace ID
+    int off = 32, len = traceIdPrefixedSpan.length - off;
+    Span.Builder span = Span.newBuilder();
+    try {
+      span.mergeFrom(traceIdPrefixedSpan, off, len);
+    } catch (IOException e) {
+      throw new AssertionError(e);
+    }
+
+    int offset = 0;
+    byte[] spanName = new byte[spanNameSize];
+    traceIdPrefix.copyTo(spanName, offset);
+    offset += traceIdPrefix.size();
+    System.arraycopy(traceIdPrefixedSpan, 0, spanName, offset, 32);
+    offset += 32;
+    SPAN_ID_PREFIX.copyTo(spanName, offset);
+    offset += SPAN_ID_PREFIX.size();
+    span.getSpanIdBytes().copyTo(spanName, offset);
+
+    span.setNameBytes(UnsafeByteOperations.unsafeWrap(spanName));
```

Comment:
weird so maybe add a comment that span name is actually the trace identifiers :P